Assets/VRTK/Source/Scripts/Interactions/Interactables/SnapAttachMechanics/VRTK_CloneSnapAttach.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Base Snap Attach|SnapAttachMechanics
```

Comment:
This needs updating to what it actually is, also there is no ordering number at the end, so it will break.